guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc

```diff
@@ -15,9 +15,25 @@ However, {Project} does not remove the back-end services on the operating system
 --foreman-proxy-tftp false
 ----
 
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets* and select a subnet.
+. Stop and mask the TFTP and DNS services (FIXME Please confirm that DNS needs to be stopped and masked too):
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# systemctl mask --now tftp.service tftp.socket systemd-resolved.service
```

Comment:
Why mask `systemd-resolved`? That looks really invasive because it's a service we never mention in our documentation or code.

Perhaps you're confusing it with the DNS service `named`?

There is also the `dhcpd` service for DHCP that you don't mention.